l10n_br_delivery_nfe/models/product_product.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2024 Diego Paradeda - KMEE
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    product_nfe40_esp = fields.Char(
+        string="Type of transported volumes",
```

Comment:
Essa string está muito longa na visão. Que tal simplificá-la para "Volume Type" e colocar a descrição completa "Type of transported volumes" no campo de ajuda?